plugins/easyimage/styles/easyimage.css

```diff
@@ -34,7 +34,14 @@ The outline is not a part of the element's dimensions, we have to use a border a
 	See https://github.com/ckeditor/ckeditor-dev/pull/1150#discussion_r150415261 for more details.
 	*/
 	float: right;
-	max-width: 25%;
+	max-width: 25%
+}
+
+/*
+Overwrite 25% in classic editor to prevent existing this bug https://bugs.chromium.org/p/chromium/issues/detail?id=803045
+*/
+body.cke_editable .cke_widget_wrapper_easyimage-side {
+	max-width: 25vw;
```

Comment:
There are some problems with `vw` unit. It was discussed during #1492 fix (see TP ticket for details). In #1492 we decided to go with `max-width: %` and `min-width: em`, but I'm not sure if it helps in this case (you may test it on #1492).

The problem with `vw` is that the image will not scale if placed inside some other container (like table), so maybe the way to go is to apply `vw` only if image is a direct body child and use `%` in other cases? WDYT?